content/tutorials/spatial_transform_crs_2/index.Rmd

```diff
@@ -0,0 +1,495 @@
+
+Here too, time to render differs between:
+
+```{r mapview_munic_heavy, eval=FALSE}
+mapview(municipalities_31370, legend = FALSE)
+```
+
+and:
+
+```{r mapview_munic_multicolour}
+mapview(comm_simpl_31370, legend = FALSE)
+```
+
+In this case, `mapview` is using the `leaflet` package with some nice defaults, but in the above examples it makes a color map by default because there's only one attribute.
+We don't want this and we make some further adjustments:
+
+```{r mapview_munic_pink}
+mapview(comm_simpl_31370, 
+        col.regions = "pink", 
+        alpha.regions = 0.3, 
+        legend = FALSE,
+        map.types = "OpenStreetMap")
+```
+
+Note that a separate tutorial about `leaflet` is [available](../spatial_create_leaflet_map/) on this website!
+
+Beware that `mapview` and `leaflet` produce maps in the '`r st_crs(3857)$Name`' CRS (`EPSG:3857`).
+For that to happen, `mapview` transforms spatial objects on-the-fly.
+[Pseudo Mercator or Web Mercator](https://proj.org/operations/projections/webmerc.html) is a much used projection in web mapping because the coordinate projection is computed much faster.
+However it should not be used in official mapping or as a projected CRS for geoprocessing, since it has more distortions (including shape) than e.g. the (truly) conformal Mercator projection (e.g. in CRS `EPSG:3395` -- `r st_crs(3395)$Name` -- which is best used in the equatorial region).
+
+Let's show two layers with `mapview`:
+
+```{r mapview_munic_water}
+mapview(list(comm_simpl_31370, watercourses), 
+        col.regions = c("pink", "blue"), 
+        alpha.regions = c(0.2, 0.5),
+        color = c("grey", NA), 
+        legend = FALSE,
+        map.types = "OpenStreetMap")
+```
+
+## Different transformation pipelines
+
+We consider a layer of points that correspond to locations where characteristics were determined of Flemish surfacewater bodies:
+
+```{r}
+path_points <- file.path(gisclubdata_path, "meetplaatsen_oppwater.shp")
+points <- read_sf(path_points)
+```
+
+Check the CRS:
+
+```{r}
+st_crs(points)
+```
+
+The `points` object has the projected '`r st_crs(points)$Name`' CRS (`EPSG:28992`), which was created for The Netherlands -- onshore.
+See e.g.
+[Wikipedia](https://nl.wikipedia.org/wiki/Rijksdriehoeksco%C3%B6rdinaten) for more background about this CRS.
+
+Essentially we are dealing with a different projected CRS than `EPSG:31370`.
+`EPSG:28992` uses the 'Amersfoort' geodetic datum, defined using the 'Bessel 1841' ellipsoid, while `EPSG:31370` has the 'Reseau National Belge 1972' datum, defined using the 'International 1924' ellipsoid [^1].
+
+[^1]: Note that several European countries, including Belgium, now provide national projected CRSs that use the common ETRS89 datum (European Terrestrial Reference System 1989).
+    An example is the Belgian CRS `EPSG:3812` (`r st_crs(3812)$Name`).
+
+We will transform `points` (back) to `EPSG:31370` since the data are outside the intended area of usage.
```

Comment:
Good to include this information about area of usage (was missing in my presentation)!